repositories.bzl

```diff
@@ -182,3 +183,12 @@ def jazzer_dependencies():
         strip_prefix = "llvm-project-jazzer-2023-04-25/compiler-rt/lib/fuzzer",
         url = "https://github.com/CodeIntelligenceTesting/llvm-project-jazzer/archive/refs/tags/2023-04-25.tar.gz",
     )
+
+    if (is_android):
+        maybe(
+            new_git_repository,
```

Comment:
`new_git_repository` is being deprecated, a regular `git_repository` should work here just as well.